utils/v2tofhirr4/mappings.bal

```diff
@@ -60,68 +60,83 @@ public function v2ToFhir(string|hl7:Message hl7) returns json|error {
 # + return - FHIR Bundle
 public function segmentToFhir(string segmentName, hl7:Segment segment) returns r4:BundleEntry[] {
     r4:BundleEntry[] entries = [];
+    V2toFhirMapperImpl impl = getMapperContext().getImpl();
     match segmentName {
         "NK1" => {
-            r4:BundleEntry entry = {'resource: nk1ToPatient(<Nk1>segment)};
-            entries.push(entry);
-            return entries;
+            Nk1ToPatient? nk1ToPatient = impl.nk1ToPatient;
+            if nk1ToPatient is Nk1ToPatient {
+                entries.push({'resource: nk1ToPatient(<Nk1>segment)});
```

Comment:
There are lots of casts.. better if we can handle them all in a separate PR may be.

Reply:
Fixed the pointed code segments. Will apply the package wide change with a seperate PR